docker-compose.demo.yml

```diff
@@ -79,13 +79,6 @@ services:
         ports:
             - "8080:8080"
 
-    mender-elasticsearch:
-        environment:
-            - ES_NETWORK_HOST=0.0.0.0
-            - ES_JAVA_OPTS=-Xms0m -Xmx100m
-        volumes:
-            - ./conductor/elasticsearch/elasticsearch.yml:/usr/share/elasticsearch/config/elasticsearch.yml
```

Comment:
I think we need to remove this same volume from the `prod.yml.template`, [here](https://github.com/mendersoftware/integration/blob/master/production/config/prod.yml.template#L132).

Reply:
Right, force-pushed to remove this volume as well.